### ISSUE TITLE
Use official CNA names to identify vulnerability sources

### ISSUE THREAD

**Issue by nscuro** [DependencyTrack/hyades#1297]
### Current Behavior

Currently, vulnerability sources are identified based on the [`Vulnerability.Source` enum](https://github.com/DependencyTrack/dependency-track/blob/master/src/main/java/org/dependencytrack/model/Vulnerability.java#L102-L112).

### Proposed Behavior

It would be better to use official CNA names instead, where applicable:

https://www.cve.org/PartnerInformation/ListofPartners

As identified in https://github.com/DependencyTrack/hyades/issues/1295, `GITHUB` should ideally be either `GitHub_P` or `GitHub_M`.

### Checklist

- [X] I have read and understand the [contributing guidelines](https://github.com/DependencyTrack/hyades/blob/main/CONTRIBUTING.md#filing-issues)
- [X] I have checked the [existing issues](https://github.com/DependencyTrack/hyades/issues) for whether this enhancement was already requested

**Comment by sahibamittal** [comment]
Needs more research and clarification. CNA partner list does not include the sources clearly and the hardcoded list in the script is missing some of names like ossindex, retire.js, npm etc.

**Comment by nscuro** [comment]
Yeah this won't fly. CNA is CVE-centric and doesn't apply to other identifiers.
